src/components/Books.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Form from './Form';
+
+export default function Books(Props) {
+  const { title, author } = Props;
+  return (
+    <div className="container">
+      <div className="book-arrange">
+        <p>
+          {title}
+        </p>
+        <p><small>{author}</small></p>
+      </div>
+      <button type="button" className="remove-btn">Remove</button>
+      <Form />
+    </div>
+  );
+}
```

Comment:
- [ ] This component represents a single book, it would make more sense to name it `Book`
- [ ] In the same way, you are currently missing the `BookList` component that renders a list of  `Book` components and the form by using the `Form` component. Let's fix that

Reply:
Required changes done